ec2/spark_ec2.py

```diff
@@ -983,6 +969,12 @@ def real_main():
     (opts, action, cluster_name) = parse_args()
 
     # Input parameter validation
+    if "." in opts.spark_version:
```

Comment:
I moved this validation down here because it's very frustrating to wait upwards of 6 minutes for a cluster to launch before finding out that you fat-fingered the version of Spark you wanted.